gbpservice/neutron/services/servicechain/plugins/ncp/node_drivers/oc_service_manager_client.py

```diff
@@ -0,0 +1,123 @@
+# One Convergence, Inc. CONFIDENTIAL
```

Comment:
This is a re-usable client library and as such should be in OC service controller repo. Not in OC  gbp repo